app/executor/executor.py

```diff
@@ -27,12 +27,13 @@ def execute(command_id):
         otherwise.
     """
     command_id = str(command_id)
-    command, arguments = databox.get_command_and_arguments(command_id)
-    if command is None:
+    gasp = databox.get_command_and_arguments(command_id)
```

Comment:
when function cannot find command it returns None, but we wanted two returns and it raised error.